model/Xsc/Event/StartScanRequest.ts

```diff
@@ -0,0 +1,15 @@
+import { ScanEventType, ScanEventStatus } from './index';
+
+export interface StartScanRequest {
```

Comment:
why are all the attributes optional? event_type,event_status,product,is_default_config... 
sound like values that always have a value